src/utils/loadGA.js

```diff
@@ -1,11 +1,19 @@
-export default function (options) {
+import memoize from '../utils/memoize';
+
+function optionToGaAddress(options) {
   let gaAddress = 'https://www.google-analytics.com/analytics.js';
   if (options && options.gaAddress) {
     gaAddress = options.gaAddress;
   } else if (options && options.debug) {
     gaAddress = 'https://www.google-analytics.com/analytics_debug.js';
   }
 
+  return gaAddress;
+}
+
+function loadGA(options) {
+  const gaAddress = optionToGaAddress(options);
+
```

Comment:
I don't think we need a memoize function, we only support one `ga` being loaded so just doing a boolean here would be fine.
```
let isLoaded = false;
function loadGA(options) {
  if (isLoaded) return;
  ...
```

Reply:
Ah thanks, that makes this PR a lot less complicated. It seems to play a lot more nicely with the test suite as well.